lib/streamlit/elements/markdown.py

```diff
@@ -66,10 +66,6 @@ def markdown(self, body, unsafe_allow_html=False):
         -------
         >>> st.markdown('Streamlit is **_really_ cool**.')
 
-        .. output::
```

Comment:
The markdown examples are also replaced by static images, or?

Reply:
@LukasMasuch Yup! The one markdown example will get replaced by this static image 
![image](https://user-images.githubusercontent.com/20672874/151005204-b9e24d1f-8045-4b0f-88f4-6b0745898ce5.png)
The same is true for all text elements found here: https://docs.streamlit.io/library/api-reference/text